app.rb

```diff
@@ -0,0 +1,153 @@
+require_relative './book'
+require_relative './person'
+require_relative './rental'
+require_relative './student'
+require_relative './teacher'
+
+def list_books
+  puts "******* \n List of all book \n ****************"
+  puts " No books addded \n ************" if @books.length.zero?
+  @books.each do |book|
+    puts "id: #{book.id} Title: #{book.title} Author: #{book.author}"
+  end
+  puts '****************'
+end
+
+def list_poeple
+  puts "******* \n List of all poeple \n ****************"
+  puts " No poeple is added \n ************" if @poeple.length.zero?
+  @poeple.each do |person|
+    puts "id: #{person.id} Title: #{person.name} Author: #{person.age}"
+  end
+  puts '****************'
+end
+
+def list_rental
+  puts "******* \n List of all rental of a person \n ****************"
+  puts " Noone rent a book \n ************" if @rentals.length.zero?
+  puts 'Enter id of a person'
+  person_id = gets.chomp.to_id
+
+  @rentals.each do |rental|
+    if rental.person.id == person_id
+      puts "On #{reantal.date} the book:  #{rental.book.title} is rented by #{person.age}"
+      puts '****************'
+    end
+  end
+end
+
+def create_person
+  puts "---------------------------- \n Do you want to create a \n1. Student or \n2. Teacher"
+  option = gets.chomp.to_i
+
+  case option
+  when 1
+    create_student
+  when 2
+    create_teacher
+  else
+    puts 'Invalid choice'
+  end
+end
```

Comment:
you have an issue when creating a person `teacher`  currently it shows this error

![image](https://user-images.githubusercontent.com/60612329/201486152-451a27f1-204b-416d-9c6f-8b4e7e583804.png)